apps/settings/js/modules/keyboard_context.js

```diff
@@ -130,9 +134,15 @@ define(function(require) {
         app[layout.layoutId].enabled = layout.enabled;
         return app[layout.layoutId];
       }
+
+      var layoutName = layout.inputManifest.nameL10nId ?
+        navigator.mozL10n.get(layout.inputManifest.nameL10nId) :
```

Comment:
We should stop using mozL10n.get:
https://developer.mozilla.org/en-US/Apps/Build/Localization/Localization_code_best_practices#Do_not_use_mozL10n.get